src/createAsyncThunk.ts

```diff
@@ -84,24 +83,67 @@ export function createAsyncThunk<
     }
   )
 
+  const aborted = createAction(
```

Comment:
Do we actually need a separate `aborted` action type?

Could we use `rejected`, and have `aborted: true` as part of `meta`?

Reply:
That would be an option, too. I'm fine either way - your call :)